samples/compose-browser/src/main/java/org/mozilla/samples/compose/browser/browser/BrowserScreen.kt

```diff
@@ -206,6 +209,8 @@ private fun Suggestions(
         )
     }
 
+    val keyboardController = LocalSoftwareKeyboardController.current
+
```

Comment:
It's nice this thing exists now 😀

Reply:
Interesting. 🤔 

I guess keeping focus is fine and preferred in situations with hardware keyboards, where we wouldn't want to remove focus and allow them to continue typing, even if they scroll the list? I haven't tried that though.

Reply:
For this particular case, I guess it makes no difference, you can still use the complete arrow in both cases. Just wanted to keep it in mind for similar use cases where one solution might be better to integrate than the other.